src/js/game/camera.js

```diff
@@ -653,28 +673,36 @@ export class Camera extends BasicSerializableObject {
         }
 
         const touch = event.changedTouches[0];
-        this.combinedSingleTouchStopHandler(touch.clientX, touch.clientY);
+        this.combinedSingleTouchStopHandler(touch.clientX, touch.clientY, enumMouseButton.left);
         return false;
     }
 
     /**
      * Internal touch start handler
      * @param {number} x
      * @param {number} y
+     * @param {enumMouseButton} buttonKey
      */
-    combinedSingleTouchStartHandler(x, y) {
+    combinedSingleTouchStartHandler(x, y, buttonKey) {
         const pos = new Vector(x, y);
-        if (this.downPreHandler.dispatch(pos, enumMouseButton.left) === STOP_PROPAGATION) {
+        if (this.downPreHandler.dispatch(pos, buttonKey) === STOP_PROPAGATION) {
             // Somebody else captured it
             return;
         }
 
-        this.touchPostMoveVelocity = new Vector(0, 0);
-        this.currentlyMoving = true;
-        this.lastMovingPosition = pos;
-        this.lastMovingPositionLastTick = null;
-        this.numTicksStandingStill = 0;
-        this.didMoveSinceTouchStart = false;
+        if (
+            buttonKey ===
+            (this.root.app.settings.getSetting("middleMousePan")
```

Comment:
Could you make this a getter?

E:g.

`get mousePanButton() { return ... }`

And then

`if (buttonKey === this.mousePanButton)`

Reply:
I've made the requested changes.